chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -120,7 +120,7 @@
   group "root"
   action :create
   variables(
-    needs_block_migrate: nova[:nova][:use_shared_instance_storage] ? false : true
+    needs_block_migrate: nova.nil? || (nova[:nova][:use_shared_instance_storage] ? false : true)
```

Comment:
It actually does not matter, because the template is not created because of `only_if { roles.include? "nova-controller" }` clause.

However, if `nova` is nil, the recipe fails during compile phase, which seems to ignore that `only_if` ...